packages/mdc-ripple/README.md

```diff
@@ -87,7 +101,7 @@ Mixin | Description
 `mdc-states-activated($color, $has-nested-focusable-element)` | Adds state and ripple styles for activated states using the indicated color, deciding opacities based on whether the passed color is light or dark.<br>`$has-nested-focusable-element` defaults to `false` but should be set to `true` if the component contains a focusable element (e.g. an input) under the root node.
```

Comment:
shorten these descriptions to one line...and then create new sections underneath "Basic States Mixins" which cover the additional information.

The table should be short and easy to read, and right now it has too much information. The new sections will allow users to deep dive.